cluster-provision/k8s/1.24/manifests/cni.do-not-change.yaml

```diff
@@ -3818,6 +3818,8 @@ spec:
           operator: Exists
         - key: node-role.kubernetes.io/master
           effect: NoSchedule
+        - key: node-role.kubernetes.io/control-plane
```

Comment:
we should not change this file please, this is the original manifest
changes should be added via cni.diff

Reply:
yes definitely - I will try to apply this to the cni.diff file.